broker/src/main/java/io/camunda/zeebe/broker/exporter/metrics/ExecutionLatencyMetrics.java

```diff
@@ -56,6 +56,10 @@ public void observeJobActivationTime(
         .observe(latencyInSeconds(creationTimeMs, activationTimeMs));
   }
 
+  public Histogram getJobLifeTime() {
```

Comment:
:thought_balloon: I'm not sure about exposing these just for testing. I wonder if there isn't a better way :thinking: I don't have an alternative though, just thinking out loud.

Reply:
💭 Instead of exposing the internals, we could replace it with a method that reads the recorded data so far. But that's also only useful for testing. So I don't see a real benefit to it. Alternatively, we could pass in the histogram through construction injection, but that's practically the same as this getter. I'm not sure how to really improve this